lockdown.json

```diff
@@ -691,17 +705,16 @@
     "0.2.0": "6bc028149440e570d495ab509692aa08bd779c6e"
   },
   "lockdown": {
-    "0.0.8-dev": "075c0c1852e7cf746e4e85ccab8113b381f0b6e0"
+    "0.0.8-dev": "e6213cc58f76c4f7b62bf64189462367a87ba112"
```

Comment:
I'm not sure. I tried to avoid this change, but lockdown consistently insisted this was the hash. I suspect it is for the same reason that peep hashes for github zip files broke recently.